CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ course's listing page.
 - Use css colors from cunningham instead of palette.scss
 - Improve the design of the dashboards layouts.
 - Improve the design of the dashboards sidebars.
+- Fix teacher dashbaord breadcrumb.
+- Remove unused route in teacher dashboard.
```

Comment:
Is it possible to split this commit in two ? One about the refactoring of the breadcrumb and a second in charge to remove all stuff related to unused routes ?

Reply:
I had two commit that i squash because it was way easier.
I'll resplit them.